staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -44,6 +44,9 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// Parallelism defines the amount of parallelism in algorithms for scheduling a Pods.
```

Comment:
indicate the default if unspecified, and the allowed min (and max, if we set a max)

Reply:
```
Parallelism defines the amount of parallelism in algorithms for scheduling a Pods. Default is 16
```
Does something like this look fine ?

Reply:
`Must be greater than 0. Defaults to 16`